### PR TITLE
Feat/netapp snapshot policy

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/netapp` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7153
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/modules/netapp/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@ovh-ux/manager-billing-components": "^3.1.3",
     "@ovh-ux/manager-catalog-price": "^1.4.1",
     "@ovh-ux/manager-components": "^1.0.0",
+    "@ovh-ux/manager-config": "^6.1.0",
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply packages/manager/modules/netapp/package.json:24]: Yes, used here: https://github.com/ovh/manager/pull/5842/files#diff-926908d1461e9bad9c326ca69096aeb513ebd0fc90e84a281ab737f14d149df1R2. I'll move this on "dependencies".

File: packages/manager/modules/netapp/src/components/snapshot-policies/controller.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+export default class SnapshotPoliciesController {
+  /* @ngInject */
+  constructor($attrs) {
+    this.$attrs = $attrs;
+    moment.locale(detectUserLocale());
+  }
+
+  $onInit() {
+    this.isSelectable = this.$attrs.selected !== undefined;
+    this.hasDeleteFunction = this.onDelete instanceof Function;
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply packages/manager/modules/netapp/src/components/snapshot-policies/controller.js:14]: Nice catch, I'll move it to `typeof`.

File: packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/constants.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const MONTHS = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+export const WEEKDAYS = [1, 2, 3, 4, 5, 6, 7];
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/constants.js:3]: No enumeration from moment, but I'll use `_range` method

File: packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/template.html [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+                data-translate="netapp_snapshot_policies_create_exemple_1"
+            ></strong>
+            <span
+                data-translate="netapp_snapshot_policies_create_exemple_1_description"
+            ></span>
+            <dl class="oui-description oui-description_horizontal my-3 w-25">
+                <dt
+                    data-translate="netapp_snapshot_policies_create_prefix"
+                ></dt>
+                <dd>My policy</dd>
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/template.html:33]: These are direct examples names. No need to get them translated

File: packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  "netapp_snapshot_policies_create_weekdays": "Jour(s) de la semaine",
+  "netapp_snapshot_policies_create_weekdays_help": "Spécifie les jours de la semaine d’exécution du programme de snapshots. Cette propriété est une liste de valeurs. La valeur minimale de cette propriété est de {{ min }} et la valeur maximale de {{ max }}.",
+  "netapp_snapshot_policies_create_snapshots_to_keep": "Copies à garder",
+  "netapp_snapshot_policies_create_name": "Nom de la Snapshot Policy",
+  "netapp_snapshot_policies_create_description": "Description de la Snapshot policy",
+  "netapp_snapshot_policies_create_empty": "vide",
+  "netapp_snapshot_policies_create_rules": "Règles (UTC Time Zone): {{ count }}",
+  "netapp_snapshot_policies_create_add_rule": "Ajouter une nouvelle règle",
+  "netapp_snapshot_policies_create_submit": "Créer nouvelle Snapshot policy",
+  "netapp_snapshot_policies_create_select_placeholder": "Choisissez...",
```
Comment (antleblanc) [inline packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json:27]:
```suggestion
  "netapp_snapshot_policies_create_select_placeholder": "Choisissez…",
```

**Review by marie-j — APPROVED** [review]

**Pull request merged**